content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -735,6 +735,7 @@ The following HTTP headers can be used to performing an impersonation request:
 * `Impersonate-User`: The username to act as.
 * `Impersonate-Group`: A group name to act as. Can be provided multiple times to set multiple groups. Optional. Requires "Impersonate-User"
 * `Impersonate-Extra-( extra name )`: A dynamic header used to associate extra fields with the user. Optional. Requires "Impersonate-User". In order to be preserved consistently, `( extra name )` should be lower-case, and any characters which aren't [legal in HTTP header labels](https://tools.ietf.org/html/rfc7230#section-3.2.6) MUST be utf8 and [percent-encoded](https://tools.ietf.org/html/rfc3986#section-2.1).
+* `Impersonate-Uid`: The UID associated with the user being impersonated. Optional. Requires "Impersonate-User"
```

Comment:
Should we explain what a UID is? We can use a tooltip for the glossary entry that covers it.
```suggestion
* `Impersonate-Uid`: The {{< glossary_tooltip text="UID" term_id="uid" >}} associated with the user being impersonated. Optional. Requires "Impersonate-User",
```

Reply:
That sounds good. I'm not familiar with the glossary tooltip, is that going to link to the uid entry in [this glossary?](https://kubernetes.io/docs/reference/glossary/?fundamental=true)

Reply:
Actually that'll link to https://k8s.io/docs/concepts/overview/working-with-objects/names (we could separately tweak that to link to https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#uids)

If you hover over the text (“UID”) you see the first part of the glossary definition.

Reply:
Here's the glossary entry that the tooltip uses: https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/uid.md